server/sockets/list.js

```diff
@@ -477,6 +477,102 @@ const changeListType = (socket, dashboardClients, cohortClients, listClients) =>
       });
   });
 
+const emitListsOnRemoveCohortMember = (socket, dashboardClients, listClients) =>
+  socket.on(CohortActionTypes.REMOVE_MEMBER_SUCCESS, data => {
+    const { cohortId, userId } = data;
+
+    List.find({ cohortId })
+      .populate('cohortId', 'memberIds')
+      .lean()
+      .exec()
+      .then(docs => {
+        if (docs) {
+          const {
+            cohortId: { memberIds: cohortMemberIds }
+          } = docs[0];
```

Comment:
Wouldn't `findOne` be a better fit for that?

Reply:
Unfortunately, in this case, I have to use `find. I have to find all cohort's lists that removed user was a viewer in. But I have to know some cohort data which is populated during querying for lists. But for all cohort's list cohort data is the same so I get it from the first list in a collection.

Reply:
Ok, however, I am still a little bit confused how the cohorts and lists work :)